bin/lean-gdb.py

```diff
@@ -1,3 +1,11 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+# Released under Apache 2.0 license as described in the file LICENSE.
+#
+# Author: Sebastian Ullrich
```

Comment:
> Well, you can already do call print(e)

That doesn't do anything for me:
```
(gdb) call print(val)
(gdb) call print(val)
```

I was thinking about casting the expr_cell to the appropriate subtype based on the kind field.  So if `m_ptr->kind == 2`, then cast to `expr_const *`.  Is this possible in gdb?